static/src/javascripts/projects/common/modules/article/space-filler.js

```diff
@@ -1,66 +1,47 @@
-/* jscs:disable disallowDanglingUnderscores */
 define([
-    'common/utils/$',
-    'common/utils/fastdom-idle',
+    'common/utils/fastdom-promise',
     'Promise',
     'raven',
     'common/modules/article/spacefinder'
 ], function (
-    $,
-    idleFastdom,
+    fastdom,
     Promise,
     raven,
     spacefinder
 ) {
-    return new SpaceFiller();
+    var lastIteration = Promise.resolve(false);
```

Comment:
Don't you find it a bit annoying to pollute the namespace and API coverage just for the sake of testing? This is sth that is direly missing in AMD, some kind of module introspection. I'll add a reset method to `fillSpace`.